crates/bevy_render/src/texture/image.rs

```diff
@@ -146,10 +148,43 @@ impl ImageSampler {
     }
 }
 
-/// Resource used as the global default image sampler for [`Image`]s with their `sampler_descriptor`
-/// set to [`ImageSampler::Default`].
+/// Global resource for [`Image`] settings.
+///
+/// Can be set via `insert_resource` during app initialization to change the default settings.
+pub struct ImageSettings {
+    /// The default image sampler to use when [`ImageSampler`] is set to `Default`.
+    pub default_sampler: wgpu::SamplerDescriptor<'static>,
+}
+
+impl Default for ImageSettings {
+    fn default() -> Self {
+        ImageSettings::default_linear()
+    }
+}
+
+impl ImageSettings {
+    /// Creates image settings with default linear sampling.
+    pub fn default_linear() -> ImageSettings {
+        ImageSettings {
+            default_sampler: ImageSampler::linear_descriptor(),
+        }
+    }
+
+    /// Creates image settings with default nearest sampling.
+    pub fn default_nearest() -> ImageSettings {
+        ImageSettings {
+            default_sampler: ImageSampler::nearest_descriptor(),
+        }
+    }
+}
+
+/// A rendering resource for the default image sampler which is set during renderer
+/// intialization.
+///
+/// The [`ImageSettings`] resource can be set during app initialization to change the default
+/// image sampler.
 #[derive(Debug, Clone, Deref, DerefMut)]
-pub struct DefaultImageSampler(pub(crate) Sampler);
+pub struct RenderDefaultImageSampler(pub(crate) Sampler);
```

Comment:
Not convinced on this rename; it makes this a breaking change and I don't think it's appreciably clearer.